modules/cluster/variables.tf

```
@@ -115,3 +115,9 @@ variable "nvidia_device_plugin" {
   default     = true
   description = "Should the nvidia device plugin be installed, required for GPU nodes"
 }
+
+variable "legacy_security_groups" {
+  type        = bool
+  default     = false
+  description = "Should we create security groups for the cluster and nodes (rather than using the cluster security group), if changed after provisioning - forces the cluster to be recreated"
```

I'm not sure I understood the wording here, is this what you mean? 
```suggestion
  description = "Should we create security groups for the cluster and nodes (rather than using the cluster security group). If changed after provisioning, it forces the cluster to be recreated"
```
Is it easier to understand like this?

---

agreed this is pretty lousy ... hopefully this is a little better!